src/Pages/Rank/RankPageInfo/RankPageInfo.js

```diff
@@ -0,0 +1,234 @@
+							<FontAwesomeIcon className='userIcon' icon={faUser} />
+							개인전
+						</IndividualPlayModeBtn>
+						<TeamPlayModeBtn playMode={playMode === 2} onClick={() => setPlayMode(2)}>
+							<FontAwesomeIcon className='usersIcon' icon={faUsers} />
+							팀전
+						</TeamPlayModeBtn>
+					</div>
+					<div className='seperateLine' />
+					<div className='rightBtnWrapper'>
+						{speedModeText.map((text, idx) => (
+							<SpeedModeBtn
+								speedMode={speedMode === idx + 1}
+								onClick={() => setSpeedMode(idx + 1)}
+								key={text + idx}
+							>
+								{text}
+							</SpeedModeBtn>
+						))}
+					</div>
+				</div>
+			</div>
+			<button className='rankingGuide' onClick={rankingGuideToggle}>
+				랭킹가이드
+			</button>
+		</RankInfoTextContainer>
+	);
+};
+
+const RankInfoTextContainer = styled.article`
+	position: absolute;
+	width: 70%;
+	height: 140px;
+	z-index: 50;
+	top: 110px;
+	left: 15%;
+	color: #fff;
+	letter-spacing: -0.7px;
+	padding: 0 10px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+
+	.monthlyRanking {
+		.textWrapper {
+			h3 {
+				font-size: 1.4rem;
+			}
+
+			div {
+				width: 35px;
+				height: 3px;
+				background-color: #fff;
+				margin: 12px 0;
+			}
+
+			p {
+				font-size: 0.75rem;
+				line-height: 1rem;
+
+				span {
+					font-weight: 600;
+				}
+			}
+		}
+
+		.btnWrapper {
+			display: flex;
+			justify-content: space-between;
+			align-items: flex-end;
+
+			.seperateLine {
+				width: 1px;
+				height: 15px;
+				background-color: #fff;
+				margin-right: 12px;
+				margin-bottom: 5px;
+			}
+
+			div {
+				margin-top: 20px;
+				margin-right: 12px;
+				display: flex;
+
+				button:hover {
+					background-color: #fff;
+					color: #015ecc;
+				}
+			}
+
+			.rightBtnWrapper {
+				display: flex;
+
+				button {
+					width: 40px;
+					height: 25px;
+					font-size: 0.75rem;
+					display: flex;
+					justify-content: center;
+					align-items: center;
+					outline: none;
+					cursor: pointer;
+
+					&:first-child {
+						border-radius: 5px 0 0 5px;
+						border-right: none;
+					}
+
+					&:last-child {
+						border-radius: 0 5px 5px 0;
+						border-left: none;
+					}
+				}
+			}
+		}
+	}
+
+	.rankingGuide {
+		width: 70px;
+		height: 22px;
+		background-color: #015ecc;
+		color: #fff;
+		border: 1px solid #fff;
+		font-size: 0.74rem;
+		font-weight: 600;
+
+		&:hover {
+			background-color: #fff;
+			color: #015ecc;
+		}
+	}
+`;
+
+const IndividualPlayModeBtn = styled.button`
+	width: 104px;
+	height: 25px;
+	border: 1px solid #fff;
+	font-size: 0.75rem;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	outline: none;
+	cursor: pointer;
+	border-radius: 5px 0 0 5px;
+
+	${props =>
+		props.playMode
+			? css`
+					background-color: #fff;
+					color: #015ecc;
+				`
+			: css`
+					background-color: #015ecc;
+					color: #fff;
+				`};
```

Comment:
삼항연산자를 속성의 value에 적용하는게 더 효율적일 것 같습니다.
```suggestion
			? css`
					background-color: `${props.playMode ? "#fff" : "#015ecc"}`;
					color: `${props.playMode ? "#015ecc" : "#fff"}`;
				`};
```

Reply:
그렇네요! 수정했습니다.